setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup

```

Comment:
Can you explain the use of this file?
Are you going to distribute the application as a standalone executable?